src/DayColumn.js

```diff
@@ -219,8 +221,10 @@ class DayColumn extends React.Component {
 
       let { height, top, width, xOffset } = style
 
+      let _draggable = get(event, draggableAccessor)
       let wrapperProps = {
         event,
+        draggable: _draggable === null ? true : _draggable,
```

Comment:
I didn't notice this before sorry, but logic specific to the DnD addon should live completely in the addons code and not here or in Event. Instead you want to add the `draggableAccessor` to context in DraggableCalendar, like we do for the other accessors: https://github.com/dbuchet-bandc/react-big-calendar/blob/f2c9cf454b795064076b879c272192897ab2b4ec/src/addons/dragAndDrop/withDragAndDrop.js#L95 and then use it off of context in the DnD specific wrapper components

Reply:
I there,
Sorry, I'm afraid i'm not familiar enough with DnD to figure how to retrieve this context in DragSource.
I've searched in DnD documentation, and I don't see how we can get context before a `beginDrag` (which occurs too late). CanDrag doesn't allow to retrieve `Component` (so its context)

Thanks for your time anyway!

Reply:
ah looked a bit more closely i see how `canDrag` is like the only method that doesn't provide the component :/

It looks like we need to wrap the DragSource in another component that injects the draggable from context...

Reply:
Yes that's what I tought too.
I'll check what I can do :)

Reply:
Mmmmmh, in fact, I don't know why I wanted to have context in DragSource... if event is not draggable, I should not return a connected Wrapper, so I just have to get context in DraggableEventWrapper.
I'll do some test and submit an update